forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-mock-store/forgerock-openbanking-uk-aspsp-rs-mock-store-server/src/main/java/com/forgerock/openbanking/aspsp/rs/store/api/openbanking/data/DataEventsApi.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2019 ForgeRock AS.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.forgerock.openbanking.aspsp.rs.store.api.openbanking.data;
+
+import com.forgerock.openbanking.common.model.data.FRDataEvent;
+import com.forgerock.openbanking.common.model.openbanking.forgerock.event.FREventNotification;
+import com.forgerock.openbanking.exceptions.OBErrorResponseException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.Collection;
+
+@RequestMapping("/api/data")
+public interface DataEventsApi {
+
+    @RequestMapping(value = "/events", method = RequestMethod.POST)
```

Comment:
Do we not need the same documentation annotations (e.g. `@ApiParam`) that are in the other `DataEventsApi ` interface above? They are on the interfaces in both the rs-gateway and rs-store for the payment side of things. I think the latter is what enabled Ben to create `openbanking-v3.1.2.json` etc within openbanking-aspsp.

Reply:
No isn't necessary, because the documentation generated it's related with the openbanking specifications and the API `/api/data/*` are helpers to feed the system for the customers and we give to them the documentation of these apis by other way for example by postman collections.

But it is a good point to discuss by the team whether such documentation will be necessary for clients.

Reply:
I think it would be useful to document it in the future, i.e. for FDX, but not necessary for OB imho